### ISSUE TITLE
selectEx multiSelect does not work

### ISSUE THREAD

**Issue by zoul0813** [vue-generators/vue-form-generator#344]
The selectEx `multiSelect` option does not appear to work as expected.  When passed a model with an array of values, the selectEx field properly selects the values in the dropdown, but when adding/removing items from the selection the field provides a single value (which appears to be the first selected item in the list).

Here's a JSFiddle, https://jsfiddle.net/zoul0813/bhvpv4ah/2/

In the JSFiddle, the selected values "a, b, e" are displayed and selected, but when you add another item to the selection or remove an item from the selection the models `features` array becomes a string.

**Comment by bary12** [comment]
I am facing the same issue. I fixed it locally by removing `v-model="value"` and adding

```js
.on('changed.bs.select', () => {
  this.value = $(this.$el).val() || [];
});
```

after initalizing the selectpicker in the mounted method.
I can open a pull request for it next week.

**Comment by zoul0813** [comment]
Closing - this appears to be resolved as of v2.3.3
